UKHomeOffice/nmsw-ui
# NMSW-470 p4 Refactor get & use declarationId from params - passenger pages

# Ticket

NMSW-470

----

## AC

see https://github.com/UKHomeOffice/nmsw-ui/pull/269

----

## To test

- create a voyage report & upload a fal 1 file so you're on task details
- click the passengers link
- > you should be taken to the do you have passengers page with the declarationId in the url
- select no and save
- > you should be returned to your task details page
- click the passengers link
- select yes and save
- > you should be taken to the upload passengers page with the declarationId in the url
- add a passengers file (NOTE: we don't have error checking for this yet)
- click check for errors
- you should be taken to the confirmation of upload page
- click save and continue
- you should be returned to the task details page

----

## Notes
The following pages will not work as they've not yet been updated

- supporting docs
- CYA
- delete
